app/views/govwales/_layout_header.html.erb

```diff
@@ -1,6 +1,6 @@
 <header id="wg_header" class="wg_header">
   <div class="header" id="header">
-    <div class="header__components">
+    <div class="header__components container">
 
       <div class="header__components__brand" id="wg_brand">
         <a href="https://gov.wales/" title="Welsh Government" class="header__brand" id="wg_logo">
```

Comment:
Need a translator here for Welsh Government title. <%= sanitize(t('header.logo.title')) %>

Reply:
And the URL

Reply:
I'll pick these up in a separate branch @markheathy

Reply:
👍